src/morphology/english/getAdjectiveForms.js

```diff
@@ -80,8 +80,8 @@ const endsWithEst = function( word ) {
  */
 const endsWithLy = function( word ) {
 	const wordLength = word.length;
-	// Consider only words of four letters or more to be adjectives (otherwise, words like "lily" are being treated as adjectives).
-	if ( wordLength > 3 ) {
+	// Consider only words of five letters or more to be adjectives (otherwise, words like "lily" are being treated as adjectives).
+	if ( wordLength > 4 ) {
```

Comment:
Small suggestion. This would keep it consistent with the comment (`five letters or more`). 
```suggestion
	if ( wordLength >= 5 ) {
```

Reply:
👍